capitalone/rubicon-ml
# Allow multiple scores and fits on an Scikit-learn pipeline

closes: #176

---

## What
  * When a `.score()` is called on a Scikit-learn pipeline with rubicon a new experiment is opened.  This allows users to log multiple scores. 
  * When a `.fit()` is called on a Scikit-learn pipeline with rubicon a new experiment is opened .  This allows users to log multiple fits. 

## How to Test
  * ensure that all the tests still run with `pytest`
  * ensure that the `Scikit-learn integration` notebook still runs without errors
